samples/subsys/usb/mass/Kconfig

```diff
@@ -9,4 +9,85 @@ config APP_WIPE_STORAGE
 	help
 	  Use this to force an existing file system to be created.
 
+choice
+	prompt "Storage and file system type used by the application"
+	default MSC_APP_STORAGE_NONE
```

Comment:
I prefer `APP_` as the prefix for application-specific Kconfig, as anything else is hard to distinguish from a non-application option.  `MSC_APP_` doesn't add much IMO.  Not a blocking request.

Reply:
Agree, it should start with APP_, fixed